rustls-cert-gen/src/cert.rs

```diff
@@ -205,6 +205,7 @@ impl EndEntityBuilder {
 /// Supported Keypair Algorithms
 #[derive(Clone, Copy, Debug, Default, Bpaf, PartialEq)]
 pub enum KeyPairAlgorithm {
+	Rsa,
```

Comment:
Should this enum variant maybe carry `RsaKeySize` now that the `rcgen` side supports specifying key size?

Reply:
Maybe, yeah, happy to see followup PRs for it.